js/src/data/types.js

```diff
@@ -0,0 +1,16 @@
+/**
```

Comment:
Since there is no consensus on where the type definitions should be placed, and there are currently scattered in many places, especially actions.js and selectors.js have accumulated many.

Given that types often need to be shared across different directories and files, I would like to centralize the type definitions of the data fetched from APIs in this file instead of adding them to actions.js or selectors.js.